docs/user/guides/build/index.rst

```diff
@@ -1,11 +1,14 @@
 How-to guides: build process
 ============================
 
-⏩️ :doc:`Setup build notifications and webhooks </guides/build-notifications>`
-    Build notifications can alert you when your builds fail so you can take immediate action.
-    In this guide,
-    you will learn how to get notified through various available channel integrations,
-    including email and chat.
+⏩️ :doc:`Enable email notifications </guides/build/email-notifications>`
+    Email notifications can alert you when your builds fail.
+    This is the most simple way to monitor your documentation builds,
+    it only requires you to switch it on.
+
+⏩️ :doc:`Setup webhook notifications </guides/build/webhooks>`
+    Webhook notifications can alert you when your builds fail so you can take immediate action.
+    We show examples of how to use the webhooks on popular platforms like Slack and Discord.
```

Comment:
I chose to call it "ENABLE email notifications", since it's just a simple operation. I think it makes the how-to more inviting.

Reply:
I think it should use setup as well right? I think you misunderstood how email notifications work.

Reply:
Ah yeah, I didn't even look at the page :face_with_head_bandage: